wrappers/src/shared_realm_cs.hpp

```diff
@@ -76,6 +82,9 @@ struct SyncConfiguration
     SchemaMode schema_mode;
 
     bool is_flexible_sync;
+
+    ClientResyncMode client_resync_mode;
+    void* managed_sync_configuration_base_handle;
```

Comment:
Same - don't need the entire type here.
```suggestion
    void* managed_sync_config;
```